fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/synonym/SynonymAnalyzer.java

```diff
@@ -108,26 +82,8 @@ protected TokenStream normalize(String fieldName, TokenStream in) {
     }
 
     @Nonnull
-    private SynonymMap getSynonymMap() {
-        return synonymMap;
-    }
-
-    private SynonymMap buildSynonymMap(final InputStreamReader synonymStream) {
-        try {
-            SynonymMap.Parser parser = new SolrSynonymParser(true, true, new Analyzer() {
-                @Override
-                protected TokenStreamComponents createComponents(String fieldName) {
-                    final StandardTokenizer src = new StandardTokenizer();
-                    TokenStream tok = new LowerCaseFilter(src);
-                    tok = new FlattenGraphFilter(tok);
-                    return new TokenStreamComponents(src, tok);
-                }
-            });
-            parser.parse(synonymStream);
-            return parser.build();
-        } catch (IOException | ParseException ex) {
-            throw new RecordCoreException("Failed to parse wordnet for synonym analyzer", ex);
-        }
+    private org.apache.lucene.analysis.synonym.SynonymMap getSynonymMap() {
```

Comment:
Does this class need to be fully specified (rather than `import`ed)?

Reply:
Fixed